kbase-extension/kbase_templates/notebook.html

```diff
@@ -14,15 +14,13 @@
 </script>
 
 <link rel="stylesheet" href="{{ static_url("components/bootstrap-tour/build/css/bootstrap-tour.min.css") }}" type="text/css" />
-<link rel="stylesheet" href="{{ static_url("components/codemirror/lib/codemirror.css") }}">
+<link rel="stylesheet" href="{{ static_url("components/codemirror/lib/codemirror.css") }}" type="text/css" />
 {{super()}}
+<link rel="stylesheet" href=""  id="kernel-css" type="text/css" />
 <link rel="stylesheet" href="{{ static_url("ext_components/datatables/media/css/dataTables.bootstrap.min.css") }}" type="text/css" />
 <link rel="stylesheet" href="{{ static_url("ext_components/select2/dist/css/select2.min.css") }}" type="text/css" />
 <link rel="stylesheet" href="{{ static_url("ext_components/select2-bootstrap-theme/dist/select2-bootstrap.min.css") }}" type="text/css" />
 <link rel="stylesheet" href="{{ static_url("ext_components/dropzone/dist/min/dropzone.min.css") }}" type="text/css" />
-
-<link rel="stylesheet" href="{{ static_url("notebook/css/override.css") }}" type="text/css" />
```

Comment:
`override.css` only contains one rule, so it was not clear why it was in a special place, especially since we already have the file `custom.css` that does a similar job. I have copied the content of `override.css` into `custom.css` and removed the reference to `override.css`

Reply:
This is duplicated in the #1906, so removing from here.